sharded_queue/api.lua

```diff
@@ -43,6 +43,11 @@ local function validate_options(options)
 end
 
 local function log_task(op_name, task)
+    if task == nil then
```

Comment:
I'd rather not duplicate the log message and use more idiomatic `type(task) == 'table' and task[1] or task`. The expression may be put in a variable

Reply:
Would mind checking other places with logging for the same behavior?

Reply:
Sure, in that case we have to change the formatter to '%s' since '%d' would throw error with nil values

Reply:
If you look at other places: https://github.com/tarantool/sharded-queue/blob/master/sharded_queue/drivers/fifottl.lua#L29

It's done pretty much like the way I did it.
Are you stil suggesting that we do this instead: `log.info(string.format([[Router[%d] %s: task %s]], fiber.self():id(), op_name, type(task) == 'table' and task[1] or task))`

Reply:
I would like to decrease the verbosity and change other places as well. Btw for not making lines too long, putting the expression into a variable seems to look cleaner. Also all messages should follow the same format for nil tasks